app-native/build.gradle

```diff
@@ -37,7 +37,7 @@ android {
             path file('src/main/cpp/CMakeLists.txt')
         }
     }
-    ndkVersion '21.0.6113669'
```

Comment:
please open a different pr for this

Reply:
Otherwise I can't do a 
<img width="218" alt="image" src="https://github.com/Countly/countly-sdk-android/assets/3314607/5053335d-9e19-4d39-bc7b-dfb2ed40e8f2">

Reply:
Ok, let us discuss this. I will write back here after discussion